squidDrone/src/imu/imu_interface.hpp

```diff
@@ -0,0 +1,87 @@
+#ifndef SRC_IMU_INTERFACE_HPP_
+#define SRC_IMU_INTERFACE_HPP_
+
+#include "i2c_handler.hpp"
+#include "basic_types.hpp"
+#include "gyroscope_sensitivity.hpp"
+#include "accelerometer_sensitivity.hpp"
+#include "mpu9250.hpp"
+
+namespace imu {
+
+  ///
+  /// @class InertialMeasurementInterface 
+  /// @brief The InertialMeasurementInterface provides an interface for the Inertial Measurement Unit.
+  ///
+  class InertialMeasurementInterface {
```

Comment:
Can setting a sensitivity, fail? If yes, than these cases have not been handled, because there is no error handling.

Reply:
I think it is not really possible to fail. Otherwise I will add this later.

Reply:
Ok